scripts/asm-installer/tests/common.sh

```diff
@@ -710,7 +714,8 @@ create_workload_service_account() {
 }
 
 create_new_instance_template() {
-  INSTANCE_TEMPLATE_NAME="vm-${LT_NAMESPACE}"
+  local SOURCE_INSTANCE_TEMPLATE=${1:-"${SOURCE_INSTANCE_TEMPLATE_NAME}"}
```

Comment:
Either make the function take arguments or directly use the env var in the test. Mixing them makes it hard to understand the actual behavior when the function is being called.